rails-commenting.rb

```diff
@@ -54,15 +54,15 @@ def destroy
     if @post.destroy
       redirect_to blog_posts_path
     else
-      # ---8)
+      # ---8) If blogpost is not deleted it goes back to blog_post_path.
       redirect_to blog_post_path(@post)
     end
   end
 
-  # ---9)
+  # ---9)Adds protection to your backend app
   private
   def blog_post_params
-    # ---10)
+    # ---10) This takes in a table and provide attributes which should be permitted.
```

Comment:
Look at the name of the method. Where else do you see the name? How would you define line 66?